src/SUMMARY.md

```diff
@@ -245,39 +245,39 @@
   - [Exercise](exercises/chromium/testing.md)
 - [Interoperability with C++](chromium/interoperability-with-cpp.md)
-  - [Example bindings](chromium/interoperability-with-cpp/example-bindings.md)
+  - [Example Bindings](chromium/interoperability-with-cpp/example-bindings.md)
   - [Limitations of CXX](chromium/interoperability-with-cpp/limitations-of-cxx.md)
-  - [CXX error handling](chromium/interoperability-with-cpp/error-handling.md)
-      - [Error handling: QR example](chromium/interoperability-with-cpp/error-handling-qr.md)
-      - [Error handling: PNG example](chromium/interoperability-with-cpp/error-handling-png.md)
+  - [CXX Error Handling](chromium/interoperability-with-cpp/error-handling.md)
+      - [Error Handling: QR Example](chromium/interoperability-with-cpp/error-handling-qr.md)
+      - [Error Handling: PNG Example](chromium/interoperability-with-cpp/error-handling-png.md)
   - [Using CXX in Chromium](chromium/interoperability-with-cpp/using-cxx-in-chromium.md)
   - [Exercise](exercises/chromium/interoperability-with-cpp.md)
-- [Adding third party crates](chromium/adding-third-party-crates.md)
+- [Adding Third Party Crates](chromium/adding-third-party-crates.md)
   - [Configuring Cargo.toml](chromium/adding-third-party-crates/configuring-cargo-toml.md)
-  - [Configuring gnrt_config.toml](chromium/adding-third-party-crates/configuring-gnrt-config-toml.md)
-  - [Downloading crates](chromium/adding-third-party-crates/downloading-crates.md)
-  - [Generating gn build rules](chromium/adding-third-party-crates/generating-gn-build-rules.md)
-  - [Resolving problems](chromium/adding-third-party-crates/resolving-problems.md)
-    - [Build scripts which generate code](chromium/adding-third-party-crates/resolving-problems/build-scripts-which-generate-code.md)
-    - [Build scripts which build C++ or take arbitrary actions](chromium/adding-third-party-crates/resolving-problems/build-scripts-which-take-arbitrary-actions.md)
-  - [Depending on a crate](chromium/adding-third-party-crates/depending-on-a-crate.md)
-  - [Reviews and audits](chromium/adding-third-party-crates/reviews-and-audits.md)
-  - [Checking into Chromium source code](chromium/adding-third-party-crates/checking-in.md)
-  - [Keeping crates up to date](chromium/adding-third-party-crates/keeping-up-to-date.md)
+  - [Configuring `gnrt_config.toml`](chromium/adding-third-party-crates/configuring-gnrt-config-toml.md)
+  - [Downloading Crates](chromium/adding-third-party-crates/downloading-crates.md)
+  - [Generating `gn` Build Rules](chromium/adding-third-party-crates/generating-gn-build-rules.md)
+  - [Resolving Problems](chromium/adding-third-party-crates/resolving-problems.md)
+    - [Build Scripts Which Generate Code](chromium/adding-third-party-crates/resolving-problems/build-scripts-which-generate-code.md)
+    - [Build Scripts Which Build C++ or Take Arbitrary Actions](chromium/adding-third-party-crates/resolving-problems/build-scripts-which-take-arbitrary-actions.md)
```

Comment:
You should probably shorten this title at some point, but you'll see after teaching it a few times :smile: